doc/source/whatsnew/v0.24.0.txt

```diff
@@ -12,6 +12,45 @@ New features
 
 .. _whatsnew_0240.enhancements.other:
```

Comment:
this label belongs to the title below your added content

Reply:
fixed